integration/templates/external_dns_values.go

```diff
@@ -3,5 +3,5 @@
 package templates
 
 // ExternalDNSValues values used for external-dns-app in integration test
-const ExternalDNSValues = `e2e: true
```

Comment:
`e2e` value is unnecessary since `priorityClass` is created from apptestctl directly.

Reply:
Thanks for removing. I think its better to create the priorityclass with apptestctl and then we can simplify the chart.